pandas/core/categorical.py

```diff
@@ -223,7 +223,7 @@ class Categorical(PandasObject):
 
     See also
     --------
-    pandas.api.types.CategoricalDtype
```

Comment:
does this property xref here? (CategoricalDtype is NOT in the main namespace)

Reply:
Ok, I've changed it to ``api.types.CategoricalDtype``. I had misunderstood the location.

Reply:
Looking more into it, I actually think ``CategoricalDtype`` isn't shown at all in the generated API docs, so the reference will be not work in ant case. Is this correct?

Reply:
Yeah, the problem is not that 'pandas' is there (that is perfectly OK, and in this case I even prefer to show the full path for clarity, as `api` is not a commonly used submodule of pandas), the issue is that this page does not exist, so it cannot link to it

Reply:
I'll put ``pandas.`` back in then.

Reply:
Yep. You can also add it to the api.rst pages so the link will work

Reply:
Change uploaded.

There seems to be something in the API.rst (line 649), but it mustn't work. @TomAugspurger has started looking into it, I'll wait till he figures it out. 

If I get to know what I should add to API.rst, I can add it to this PR.

Reply:
In api.rst it should be `pandas.api.type.CategoricalDtype` (missing the pandas).

But the autoclass fit the best with the rest of the autosummary's, so I may change it. For now, I'd just change your reference to `pandas.api.type.CategoricalDtype`, and I'll make sure they're valid before the release.